### PR TITLE
Feat network overview

### PR DESCRIPTION
**Pull request by nunogois**
https://linear.app/unleash/issue/2-512/exploration-network-overview-represented-as-a-flow-chart

image

### CODE REVIEW

File: frontend/src/component/admin/menu/AdminMenu.tsx [hunk 1]
```diff
@@ -11,6 +10,8 @@ function AdminMenu() {
     const { isBilling } = useInstanceStatus();
     const { flags } = uiConfig;
 
+    const activeTab = pathname.split('/')[2];
```
Comment (nunogois) [inline frontend/src/component/admin/menu/AdminMenu.tsx:13]: Simplifies the logic, grabs routes with 1 level deep sub-routes like `/network/traffic`.

File: frontend/src/component/menu/Header/Header.tsx [hunk 2]
```diff
@@ -76,7 +76,11 @@ const Header: VFC = () => {
         mobileRoutes: routes.mobileRoutes.filter(filterByConfig(uiConfig)),
         adminRoutes: routes.adminRoutes
             .filter(filterByConfig(uiConfig))
-            .filter(filterByEnterprise),
+            .filter(filterByEnterprise)
+            .map(route => ({
+                ...route,
+                path: route.path.replace('/*', ''),
```
Comment (nunogois) [inline frontend/src/component/menu/Header/Header.tsx:82]: Allows us to have wildcard `/*` paths on `routes.ts` while cleaning them up here.

File: frontend/src/hooks/api/getters/useInstanceMetrics/useInstanceMetrics.ts [hunk 3]
```diff
@@ -18,10 +18,13 @@ export interface IInstanceMetricsResponse {
...
 
-export const useInstanceMetrics = (): IInstanceMetricsResponse => {
+export const useInstanceMetrics = (
+    options: SWRConfiguration = {}
+): IInstanceMetricsResponse => {
     const { data, error, mutate } = useSWR(
         formatApiPath(`api/admin/metrics/rps`),
-        fetcher
+        fetcher,
+        options
```
Comment (nunogois) [inline frontend/src/hooks/api/getters/useInstanceMetrics/useInstanceMetrics.ts:27]: Passing in the options as a parameter allows us more flexibility if needed, like updating this based on an interval in the future, if needed. Not being used yet though, I only experimented a bit with it, so I can clean it up for now and only add it when needed.

File: src/lib/routes/admin-api/metrics.ts [hunk 4]
```diff
@@ -187,18 +187,24 @@ class MetricsController extends Controller {
...
+            const [clientMetrics, frontendMetrics, adminMetrics] =
+                await Promise.all([
+                    this.clientInstanceService.getRPSForPath(
+                        '/api/client/.*',
+                        hoursToQuery,
+                    ),
+                    this.clientInstanceService.getRPSForPath(
+                        '/api/frontend.*',
+                        hoursToQuery,
+                    ),
```
Comment (nunogois) [inline src/lib/routes/admin-api/metrics.ts:199]: Adds `frontend` (embedded proxy) requests to the metrics.

Comment (chriswk) [reply src/lib/routes/admin-api/metrics.ts:199]: Nice. Gaston I talked about it, but didn't get around to do it just yet.

**Review by chriswk — APPROVED** [review]
Looks really cool Nuno. Loved the demo :)

**Pull request merged**